js/ui/grid_core/ui.grid_core.filter_row.js

```diff
@@ -538,11 +543,15 @@ var ColumnHeadersViewFilterRowExtender = (function() {
                     that.getController("editorFactory").loseFocus();
                 },
                 onSubmenuHiding: function() {
-                    var menu = Menu.getInstance($menu);
-
                     eventsEngine.trigger($menu, "blur");
-                    menu && menu.option("focusedElement", null);
-                    isCellWasFocused && that._focusEditor($editorContainer);
+                    restoreFocus();
+                },
+                onContentReady: function(e) {
+                    eventsEngine.on($menu, "blur", () => {
+                        var menu = e.component;
+                        menu._hideSubmenu(menu._visibleSubmenu);
```

Comment:
Is it posiible not use private API of Menu widget?

Reply:
There are no public methods to close the menu. I also tried to trigger various events on the menu element, but this did not produce results.